diesel/src/pg/metadata_lookup.rs

```diff
@@ -0,0 +1,32 @@
+use std::mem;
+
+use prelude::*;
+use super::{PgConnection, PgTypeMetadata};
+
+#[derive(Debug)]
+pub struct PgMetadataLookup {
```

Comment:
This could not implement `Debug` because `PgConnection` does not implement `Debug`